### ISSUE TITLE
How to edit layout of navlinks?

### ISSUE THREAD

**Issue by paulradzkov** [lucor/docpad-plugin-navlinks#2]
For example I need to translate "Previous Post:" and "Next Post:" into another language and add some extra markup.

**Comment by andreyvolokitin** [comment]
Markup is hardcoded into plugin, so I think you need to change plugin source...

**Comment by paulradzkov** [comment]
...which will be rewritten after plugin update.

I think would be better to implement custom template functionality.

**Comment by andreyvolokitin** [comment]
For sure

**Comment by lucor** [comment]
You are right. Actually the template is hardcoded. A template functionality should be implemented. Regarding i18n seems there is not standard at moment in docpad bevry/docpad/issues/17

**Comment by RobLoach** [comment]
Could introduce a partial for it.

**Comment by paulradzkov** [comment]
Any updates on this?

**Comment by RobLoach** [comment]
I used CSS to change the look of it as needed, but I'm sure there would be a better solution around it. One thing we could do is introduce a partial use like what docpad-plugin-menu does:
https://github.com/sergeche/docpad-plugin-menu

Would need a PR though.

**Comment by paulradzkov** [comment]
@RobLoach Partial would be great!

**Comment by jasonb-vbt** [comment]
+1. I had to hack the source to change the link line too. This should be configurable from your docpad.coffee file.

A partial would be ok, but I'd much prefer just to supply a `plugins.navlinks.renderLinks` function in my docpad.coffee file.  This function would:
-  get called from inside the `getNavLinks()` function,
- take the `navlinks` object as input
- return whatever HTML you want to see in the output
